src/pages/hybrid-homepage/hybrid-homepage.module.scss

```diff
@@ -78,7 +78,11 @@
 
 //sections:
 .landingSection {
-	padding-top: 35px
+	padding-top: 35px;
+	width: calc(100% - 40px);
```

Comment:
why not just add padding from the sides?

Reply:
done